cmd/wallet-web/src/pages/WACIShare.vue

```diff
@@ -5,7 +5,7 @@
 -->
 
 <template>
-  <div class="w-full flex justify-center items-start">
+  <div class="flex justify-center items-start w-full">
```

Comment:
was this updated automatically ?

Reply:
If not, can we ignore this file from check-in ?

Reply:
yes

Reply:
@birtony

Reply:
ignore this file @Baha-sk